src/gc/gcpriv.h

```diff
@@ -4302,16 +4302,22 @@ dynamic_data* gc_heap::dynamic_data_of (int gen_number)
     return &dynamic_data_table [ gen_number ];
 }
 
+#ifdef FEATURE_PAL
```

Comment:
This should not be under `FEATURE_PAL`. We want to avoid having `FEATURE_PAL` in the GC implementation.